### PR TITLE
Thrift Client Configuration Modification

### PR DESCRIPTION
**Pull request by Thanu**
This P/R includes Stratos thrift-client-config.xml file change to read DAS thrift client configuration from the same file and thrift config parser and publisher changes for the thrift-client-config.xml modification.. DAS configuration read from this file will be used in CC and AS to publish thrift events.

### CODE REVIEW

File: components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/wso2/cep/WSO2CEPHealthStatisticsPublisher.java [hunk 1]
```diff
@@ -37,12 +37,12 @@
 
     private static final Log log = LogFactory.getLog(WSO2CEPHealthStatisticsPublisher.class);
 
-    private static final String STATS_PUBLISHER_ENABLED = "cep.stats.publisher.enabled";
     private static final String DATA_STREAM_NAME = "cartridge_agent_health_stats";
     private static final String VERSION = "1.0.0";
+    public static final String CEP_THRIFT_CLIENT_NAME = "cep";
```
Comment (swgkg) [inline components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/wso2/cep/WSO2CEPHealthStatisticsPublisher.java]: Do we need to make this as public?

File: components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/wso2/cep/WSO2CEPInFlightRequestPublisher.java [hunk 2]
```diff
@@ -39,12 +39,12 @@
 public class WSO2CEPInFlightRequestPublisher extends ThriftStatisticsPublisher implements InFlightRequestPublisher {
     private static final Log log = LogFactory.getLog(WSO2CEPInFlightRequestPublisher.class);
 
-    private static final String STATS_PUBLISHER_ENABLED = "cep.stats.publisher.enabled";
     private static final String DATA_STREAM_NAME = "in_flight_requests";
     private static final String VERSION = "1.0.0";
+    public static final String CEP_THRIFT_CLIENT_NAME = "cep";
```
Comment (swgkg) [inline components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/wso2/cep/WSO2CEPInFlightRequestPublisher.java]: Do we need to make this as public?

Comment (Thanu) [inline components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/wso2/cep/WSO2CEPInFlightRequestPublisher.java]: Fixed it with commit 857e308ea470c0dd857122c6f870c3ffba82aab3.

File: components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/wso2/cep/WSO2CEPHealthStatisticsPublisher.java (hunk 1 above)
Comment (Thanu) [inline components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/wso2/cep/WSO2CEPHealthStatisticsPublisher.java]: Fixed it with commit 857e308ea470c0dd857122c6f870c3ffba82aab3.

**Pull request closed without merging**